asdf_astropy/resources/schemas/fits/fitswcs-1.0.0.yaml

```diff
@@ -0,0 +1,17 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://astropy.org/schemas/astropy/fits/fitswcs-1.0.0"
+
+title:
+  Represents the fits object
+
+description:
+  Represents the fits object
+
+allOf:
+  - tag: "tag:astropy.org:astropy/fits/fitswcs-1.0.0"
+  - type: object
+    properties:
+      header:
+        type: object
```

Comment:
Can this be made more specific? Having a generic `object` will pass just about anything.